collect_app/src/main/java/org/odk/collect/android/tasks/FormLoaderTask.java

```diff
@@ -254,6 +256,18 @@ private FormDef createFormDefFromCacheOrXml(String formPath, File formXml) {
         return null;
     }
 
+    private static String getMediaPath(String formFilename) {
+        String regexString = "(.*)\\/(.*)\\.xml";
```

Comment:
I think `(.+?)` would be more correct/explicit since:
- we don't want empty filenames
- we need the shortest possible string after the last slash. This is already happening because of the order of application of groups (the first group will take everything it can take), but making it explicit lifts part of the effort of understanding the regexp.

Reply:
I agree. Thanks.
I wondered whether Collect allows form file extensions other than .xml, or files with no extensions, and decided to also support those alternatives. The regex got more complicated than I liked, so I rewrote it as you see it now, with a bit of `String.lastInstanceOf`.  What do you think?